flexmeasures/cli/tests/conftest.py

```diff
@@ -96,3 +96,41 @@ def reporter_config_raw(app, db, setup_dummy_data):
     )
 
     return reporter_config_raw
+
+
+@pytest.mark.skip_github
+@pytest.fixture(scope="module")
+def shiftable_load_power_sensor(db, app):
+    """
+    Create an asset with two sensors (1 and 2), and add the same set of 200 beliefs with an hourly resolution to each of them.
```

Comment:
I don't think this line is correct at all (must be from copy/paste)

Reply:
Thanks for pointing it out :smile:

Reply:
Thanks for noticing, somehow It didn't get pushed...